### PR TITLE
Bug fix: Fixed issue with deploy scripts not honoring existing SKU se…

### PR DESCRIPTION
**Pull request by rraomsft**
Bug fix: Fixed issue with deploy scripts not honoring existing SKU settings

### CODE REVIEW

File: Common/Deployment/PrepareIoTSample.ps1 [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿Param(
     [Parameter(Mandatory=$True,Position=0)]
-    $environmentName,
+    $environmentName = "rraorm",
```
Comment (handeadiguzel) [inline Common/Deployment/PrepareIoTSample.ps1]: why this: rraorm?

Comment (aguilaaj) [inline Common/Deployment/PrepareIoTSample.ps1]: I think it's Ritesh's alias + rm? Is this intentional @rraomsft?

Comment (rraomsft) [inline Common/Deployment/PrepareIoTSample.ps1]: Fixed. Got inadvertently added during testing

File: Common/Deployment/PrepareIoTSample.ps1 [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
     [Parameter(Mandatory=$True,Position=1)]
-    $configuration,
+    $configuration = "cloud",
```
Comment (rraomsft) [inline Common/Deployment/PrepareIoTSample.ps1]: Fixed. Got inadvertently added during testing

**Comment by rraomsft** [comment]
Related to issue #342

**Pull request merged**